plot/plot_biases.py

```diff
@@ -0,0 +1,130 @@
+# Plot IMU bias data from a JSONL file whose rows have data in this kind of format:
```

Comment:
Tip: If you replace this comment with the corresponding (multi-line) string, it becomes the _docstring_ (`__doc__`), which is shown as the help text if you run it with the `--help` switch, i.e.,
```python
"""
Plot IMU bias data from a JSONL file whose rows have data in this kind of format:


   {
     "time": 0.0,
     "biasMean": {
       "GyroscopeAdditive": {
         "x": 0.0,
         "y": 0.0,
         "z": 0.0
       },
      ...
"""
import json
# ...
```

Reply:
I forgot about that. Fixed now.